config.lua

```diff
@@ -39,6 +39,25 @@ ox = {
         [0.0] = 0.00,
     },
 
+    -- Electic vehicles
+    electricModels = {
```

Comment:
Should be enough to do
`electricModels = { `model1`, `model2` ... }`

Reply:
No what he said makes sense, you'd have to loop the table otherwise and find the hash, the way he did it you can directly reference it.

Reply:
Oh my bad